Projects/Claims/Sources/Views/SubmitClaim/GlassDamage/SubmitClaimGlassDamageScreen.swift

```diff
@@ -0,0 +1,184 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct SubmitClaimGlassDamageScreen: View {
+    @State var selectedFields: [String] = []
+
+    var body: some View {
+        hForm {
+            VStack(spacing: 16) {
+                hSection {
+                    InfoCard(text: L10n.submitClaimGlassDamageInfoLabel, type: .info)
+                }
+                VStack(spacing: -8) {
+                    ClaimContactCard(
+                        icon: hCoreUIAssets.carGlass,
+                        label: L10n.submitClaimGlassDamageOnlineBookingLabel,
+                        buttonText: L10n.submitClaimGlassDamageOnlineBookingButton,
+                        title: L10n.submitClaimPartnerTitle
+                    )
+
+                    ClaimContactCard(
+                        icon: hCoreUIAssets.rydsBilglas,
+                        label: L10n.submitClaimGlassDamageOnlineBookingLabel,
+                        buttonText: L10n.submitClaimGlassDamageOnlineBookingButton
+                    )
+                }
+
+                hSection {
+                    VStack(alignment: .leading, spacing: 8) {
+                        hText(L10n.submitClaimHowItWorksTitle)
+                        hText(L10n.submitClaimGlassDamageHowItWorksLabel)
+                            .foregroundColor(hTextColorNew.secondary)
+                    }
+                }
+                .padding(.top, 8)
```

Comment:
After we remove vertical spacing inside ClaimContractCard, we should switch this to 24 as it is in design